src/sagemaker_training/pytorch_xla.py

```diff
@@ -72,6 +72,19 @@ def _setup(self):  # type: () -> None
         logger.info("Starting distributed training through PT-XLA Runtime.")
         self._check_compatibility()
 
+        # Set NCCL logging to info to debug customer issues
+        os.environ["NCCL_DEBUG"] = "info"
```

Comment:
this may potentially conflict with the settings in the docker files. We currently add this in the deep learning containers like so:
`echo NCCL_DEBUG=INFO >> /etc/nccl.conf`

Reply:
Overriding the default is the intent. We want to explicitly enable debug logging for NCCL operations. This will enable customers to potentially share more detailed logs with us when they encounter failures with Training Compiler in distributed training scenario(, thereby allowing us to debug issues faster). In our internal benchmarks we observed that we can offer competitive performance even with this debug logging turned on. Also, SageMaker data parallel also explicitly enables debug logging for NCCL operations when it's enabled.